CefSharp.Example/Resources/BindingTestAsync.js

```diff
@@ -2,209 +2,209 @@ QUnit.module('BindingTestAsync', (hooks) =>
 {
     hooks.before(async () =>
     {
-        await CefSharp.BindObjectAsync("boundAsync");
+        await CefSharp.BindObjectAsync('boundAsync');
```

Comment:
Please revert the styling change, it creates too much noise and doesn't actually provide a functional change.

Makes it very hard to review what's actually changed.

Reply:
Sorry. I did it out of habit.